packages/gestalt/src/Image.js

```diff
@@ -81,14 +81,10 @@ export default class Image extends PureComponent<Props> {
   static defaultProps: {|
     color: string,
     fit?: 'contain' | 'cover' | 'none',
-    importance?: 'high' | 'low' | 'auto',
-    loading?: 'eager' | 'lazy' | 'auto',
```

Comment:
@mziemer21 Could you explain a little bit more for context, why `loading: 'auto'` is not "auto" anymore in relationship with `importance`?